docs/api.md

```diff
@@ -1848,6 +1881,157 @@ This endpoint supports retrieving extended metadata; provide a header `X-Extende
+              <orx:meta><orx:instanceID>uuid:85cb9aff-005e-4edd-9739-dc9c1a829c44</orx:instanceID></orx:meta>
+              <name>Alice</name>
+              <age>32</age>
+            </data>
+
+### Creating a Submission [POST /v1/projects/{projectId}/forms/{xmlFormId}/submissions{?deviceID}]
+
+To create a Submission by REST rather than over the [OpenRosa interface](/reference/openrosa-endpoints/openrosa-form-submission-api), you may `POST` the Submission XML to this endpoint. The request must have an XML `Content-Type` (`text/xml` or `application/xml`).
+
+Unlike the OpenRosa Form Submission API, this interface does _not_ accept Submission attachments upon Submission creation. Instead, the server will determine which attachments are expected based on the Submission XML, and you may use the endpoints found in the following section to add the appropriate attachments and check the attachment status and content.
+
+If the XML is unparseable or there is some other input problem with your data, you will get a `400` error in response. If a submission already exists with the given `instanceId`, you will get a `409` error in response.
+
++ Parameters
+    + xmlFormId: `simple` (string, required) - The `xmlFormId` of the Form being referenced.
+    + deviceID: `b1628661-65ed-4cab-8e30-19c17fef2de0` (string, optional) - Optionally record a particular `deviceID` associated with this submission. It is recorded along with the data, but Central does nothing more with it.
+
++ Request (application/xml)
+    + Body
+
+            <data id="simple">
+              <orx:meta><orx:instanceID>uuid:85cb9aff-005e-4edd-9739-dc9c1a829c44</orx:instanceID></orx:meta>
+              <name>Alice</name>
+              <age>32</age>
+            </data>
+
++ Response 200 (application/json)
+    + Attributes (Submission)
+
++ Response 400 (application/json)
+    + Attributes (Error 400)
+
++ Response 403 (application/json)
+    + Attributes (Error 403)
+
++ Response 409 (application/json)
+    + Attributes (Error 409)
+
+### Updating Submission Data [PUT /v1/projects/{projectId}/forms/{xmlFormId}/submissions/{instanceId}]
+
+_(introduced: version 1.2)_
+
+You can use this endpoint to submit _updates_ to an existing submission.
+
+The `instanceId` that is submitted with the initial version of the submission is used permanently to reference that submission logically, which is to say the initial submission and all its subsequent versions. Each subsequent version will also provide its own `instanceId`. This `instanceId` becomes that particular version's identifier.
+
+To perform an update, you need to provide in the submission XML an additional [`deprecatedID` metadata node](https://getodk.github.io/xforms-spec/#metadata) with the `instanceID` of the particular submission version you are replacing. If the `deprecatedID` you give is anything other than the identifier of the current version of the submission at the time the server receives it, you will get a `409 Conflict` back.
```

Comment:
I'm confused about this. Does "current version of the submission" mean the initial instance id? I don't think so, but if not, how do you *get* the latest ID?

Like, if there are 3 updates to a submission:
`A`
`B`
`C`

`A` will be how you refer to that submission even when it updates... but do you have to set the deprecated ID to `B` to make the `C` update?

Reply:
I see there's more info about this in the openrosa part of the API... would you want to link from here to there?

For my above comment, I think a clarification like "current version - possibly not the initial instance ID if already deprecated" would help.

Reply:
Maybe it would be helpful to explicitly describe requesting the latest instance XML (which may or may not have the same instanceId as the API uses to reference the submission) and then as part of that edit changing the instanceId from the XML to be the deprecatedId and populating a new instanceId. That is, whatever client performs the edit will use the latest instance XML so it doesn't care what id Central uses the reference the submission by.

Reply:
I /think/ the primary confusion is "how do you get the latest ID?". I believe the answer is that you don't explicitly, you just request the latest XML and do your edits including the id switching from there.

Reply:
yeah, i understood that part.